src/NHibernate/Multi/MultiAnyLinqQuery.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using NHibernate.Linq;
+using NHibernate.Util;
+using Remotion.Linq.Parsing.ExpressionVisitors;
+
+namespace NHibernate
+{
+	public partial class MultiAnyLinqQuery<T> : MultiAnyQuery<T>
+	{
+		private static Delegate _postExecuteTransformer;
+		private static NhLinqExpression _linqEx;
+
+		public MultiAnyLinqQuery(IQueryable query):base(GetQuery(query))
```

Comment:
Non standard formatting, I guess a reformat would alter this file. Better do it before merging.